### DISCUSSION TITLE
Creating lines in documents

### DISCUSSION THREAD

**Discussion by ameresv** [quarto-dev/quarto-cli#5563] [Q&A] [answered]
Hi, guys! 
This could be sound odd, but I want to create horizontal and vertical lines by the time to render a qmd file to pdf. At this moment, I have tried using the three dashes '---' to separate, e.g. the summary section from the title; however, what I obtained is a line as is shown in the pdf attached: a partial line in the middle of the page. 
Is there a way to create a bigger one? I mean, a line that I could use as separator both vertical and horizontal? 
Thanks a lot.
Regards 
[table.pdf](https://github.com/quarto-dev/quarto-cli/files/11489252/table.pdf)

**Comment by mcanouil** [comment, marked as answer]
If `---` (_i.e._ `\begin{center}\rule{0.5\linewidth}{0.5pt}\end{center}`) does not fit your needs, then go for raw LaTeX, see <https://duckduckgo.com/?q=latex+rule+height>.

For example:
````qmd
---
title: "Quarto Playground"
format: pdf
---

## Quarto Playground 1

This is a playground for Quarto.

`\rule{\linewidth}{5cm}`{=tex}

## Quarto Playground 2
````
<img width="580" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/8e93fb5a-c8f3-4b5a-b221-f62f9b60cf80">

**Reply by ameresv** [reply]
Thanks a lot. I am using quarto CLI at ubuntu and I realized that there is a lot to learn over here. 
Regards
